bigtable-client-core-parent/bigtable-legacy-protos/README.md

```diff
@@ -0,0 +1,41 @@
+# Bigtable Protos
+
+## Purpose
+This project houses the raw .proto files that are used by bigtable-legacy-compatibility, which is primarily geared towards v1 protobuff objects used by Dataflow.  This project generates a jar with java artifacts derived from the proto files in this project.  These proto files can also be used to generate other clients.
```

Comment:
You might also want to use .proto everywhere instead of proto, but I don't feel strongly about it.